docker/config/webapp.env.template

```diff
@@ -9,3 +9,5 @@ FXA_CLIENT_ID=2651b9211a44b7b2
 FXA_SECRET_KEY=a3cafccbafe39db54f2723f8a6f804c337e362950f197b5b33050d784129d570
 FXA_OAUTH_ENDPOINT=https://oauth-stable.dev.lcip.org/v1
 FXA_PROFILE_ENDPOINT=https://stable.dev.lcip.org/profile/v1
+GITHUB_CLIENT_ID=f70b793c5bf6a66aa3d8
+GITHUB_SECRET_KEY=dd5a595346d645b25e6107288298512e92bea9e2
```

Comment:
Sorry @abowler2. As agreed with @Pike, we should actually remove those two lines, because they aren't meant to be public anyways. Users we'll need to set up their own to use GitHub OAuth, and we have that documented.

Reply:
No worries. Done and pushed.